docs/coding-projects/gas-station.md

```diff
@@ -0,0 +1,654 @@
+- You have installed the `kadena-cli` package and have a working directory with initial configuration settings.
+- You have a local development node that you can connect to that runs the `chainweb-node` program, either in a Docker container or on a physical or virtual computer.
+- You should be familiar with defining modules and using keysets.
+
+## Get the starter code
+
+To get started:
+
+1. Open a terminal shell on your computer.
+
+2. Clone the `pact-coding-projects` repository by running the following command:
+
+   ```bash
+   git clone https://github.com/kadena-docs/pact-coding-projects.git
+   ```
+
+3. Change to the `04-gas-station` directory by running the following command:
+
+   ```bash
+   cd pact-coding-projects/04-gas-station
+   ```
+
+   If you list the contents of this directory, you'll see the following files:
+
+   - `starter-gas-account.pact` provides a framework for the project code in comments.
+   - `gas-account.pact` contains the final module code for the sample gas account.
+   - `gas-account.repl` provides the complete test file for testing the final `gas-account.pact` file.
+
+4. Open and review the `starter-gas-account.pact` file.
+
+   This file describes all of the tasks that you need to complete for the _Gas station accounts_ coding project.
+   You can follow the instructions embedded in the file to try to tackle this coding project on your own
+   without looking at the solutions to each step, or follow the instructions in the next sections if you need additional guidance.
+
+## Implement the gas payer interface
+
+Because paying gas fees is a common operation, the `gas-payer-v1` interface is defined in a module that's deployed by default on the Kadena development, test, and main networks.
+For this coding project, you'll implement this interface in a new Pact module—the `test-gas` module—in a `gas-account.pact` file.
+
+To create the gas station module:
+
+1. Open the code editor on your computer.
+
+2. Create a new `gas-account.pact` file.
+
+3. Add the minimal Pact code required to define a module.
+
+   Remember that a module definition requires the following information: 
+
+   - namespace
+   - governing keyset or capability
+   - at least one function
+   
+   The function in this case is an implementation of the `gas-payer-v1` interface as defined in the `gas-payer-v1.pact` file.
+   For example:
+
+   ```pact
+   (namespace "free")
+
+   (module test-gas GOVERNANCE
+      (defcap GOVERNANCE () true)
+   
+
+      (implements gas-payer-v1)
+
+   )
+   ```
+ 
+   To deploy the gas station module, you must use an existing namespace such as the `"free"` namespace.
+   For local testing, you can create the `test-gas` module in a custom namespace that you define for the local development network.
+
+   After the `implements` keyword, the `test-gas` module must implement the capabilities, types, and functions as defined in the `gas-payer-v1.pact` file.
```

Comment:
"After th eimplements keyword" is  kind of a weird way to say this. I think it means "once an `implements` statement is issued, the implementing module must implement the ...." etc.